### DISCUSSION TITLE
Soroban Quest 7 NFT's

### DISCUSSION THREAD

**Discussion by fluffypanda7** [stellar/sorobanathon#157] [General]
Hope I made it in time!

Please let me know , Thank you

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Comment by fluffypanda7** [comment]
Hello Spintax,

Thank you for your email. I have filled out the form and submitted it.
Please let me know if anything went wrong.

Thank you again,

Helene

**Reply by stellarjohn** [reply]
Hi Helene,

We still haven't received your typeform. Please try to submit again. 
You should end up on a screen that looks like this on completion: 
![image](https://user-images.githubusercontent.com/98538609/210407012-d93b896d-358c-487a-baf1-b38d52629360.png)

**Comment by fluffypanda7** [comment]


**Reply by stellarjohn** [reply]
You have to have to confirmation in order to receive payment. Could you try to submit again and provide the number here if you are able to get it?

**Reply by stellarjohn** [reply]
Hi! Just needed the confirmation number. I highly recommend deleting this comment with your person documents.

**Reply by fluffypanda7** [reply]
Thank you! I didn't realize that.

My confirmation number is   140670026

**Comment by rawritude** [comment]
Your kyc and drivers licence got emailed to anyone subscribed to this
thread. Highly recommend at least deleting your last message.

**Reply by fluffypanda7** [reply]
Thank you. Deleted.
